Docs

fabioaletorres opened the pull request:
Att composição da equipe.

viniciusvieira00 committed:
Init Docs branch

viniciusvieira00 committed:
Theme language adjustment

BrunoBReis committed:
docs: updating yml file

BrunoBReis committed:
docs: creating an adding text to cenario-atual

BrunoBReis committed:
docs: fishbone asset

BrunoBReis committed:
docs: creating and adding texto to solucao-proposta

BrunoBReis committed:
docs: creating files for another topics

BrunoBReis committed:
docs: removing sobre page

viniciusvieira00 committed:
Merge pull request #2 from mdsreq-fga-unb/1-atualizacao-da-documentacao

1 atualizacao da documentacao

BrunoBReis committed:
style: adding css to mkdocs

BrunoBReis committed:
docs: adding docs to estrategias

BrunoBReis committed:
docs: updating date

PauloFilho2 committed:
docs: Update cenario-atual

PauloFilho2 committed:
docs: Update solucao-proposta

PauloFilho2 committed:
Merge pull request #16 from mdsreq-fga-unb/14-nova-versao-da-primeira-parte

14 nova versao da primeira parte

BrunoBReis committed:
ci: moving ci to docs branch

BrunoBReis committed:
docs: adding simple text to test

BrunoBReis committed:
Merge pull request #18 from mdsreq-fga-unb/17-ci-nao-esta-funcionando

ci: moving ci to docs branch

PauloFilho2 committed:
docs: Creating and adding text to interacao-entre-equipe-e-cliente

Stain19 committed:
Merge pull request #15 from mdsreq-fga-unb/7-33-justificativa

docs: Adicionando Escolha das Estrategias, Justificativa e quadro comparativo

viniciusvieira00 committed:
Merge with docs

PauloFilho2 committed:
docs: add tabela de versao

PauloFilho2 committed:
docs: date update

PauloFilho2 committed:
Merge pull request #19 from mdsreq-fga-unb/10-51-composição-da-equipe

docs: Creating and adding text to interacao-entre-equipe-e-cliente

viniciusvieira00 committed:
Cronograma de entregas

viniciusvieira00 committed:
ci: Retorno de arquivo previamente excluido

BrunoBReis committed:
docs: creating licoes aprendidas

fabioaletorres committed:
docs: adicao da comunicacao/validacao

fabioaletorres committed:
Merge pull request #23 from mdsreq-fga-unb/11-52-comunicação

docs: adicao da comunicacao/validacao

Stain19 committed:
Merge pull request #20 from mdsreq-fga-unb/4-cronograma-e-entregas

4 cronograma e entregas

BrunoBReis committed:
docs: fixing

Stain19 committed:
Merge pull request #22 from mdsreq-fga-unb/21-fazer-a-parte-de-licoes-aprendidas

docs: creating licoes aprendidas

BrunoBReis committed:
docs: adding meet link

viniciusvieira00 committed:
Merge pull request #25 from mdsreq-fga-unb/24-link-do-reuniao

docs: adding meet link

viniciusvieira00 committed:
docs: ajuste de requirements.txt

viniciusvieira00 committed:
docs: Ajuste mkdocs.yml

viniciusvieira00 committed:
docs: Ajustes assets

viniciusvieira00 committed:
docs: Guia de contribuição, assets e página inicial

viniciusvieira00 committed:
Ajustes de documentação e estilização

viniciusvieira00 committed:
fix: Adjust on docs

viniciusvieira00 committed:
Merge pull request #27 from mdsreq-fga-unb/feature/ajustes-documentacao

Feature/ajustes documentacao

viniciusvieira00 committed:
fix:deployment fix

viniciusvieira00 committed:
docs: add theoric references in the software engineering strategies document

viniciusvieira00 committed:
docs: remove unused content on commits policy

viniciusvieira00 committed:
docs: change in delivery schedule and sprints

viniciusvieira00 committed:
docs: remove unused text

BrunoBReis committed:
docs: updating fish bone diagram

BrunoBReis committed:
docs: removing deprecated image

BrunoBReis committed:
docs: adding new image

BrunoBReis committed:
docs: fixing name with kebab-case

BrunoBReis committed:
docs: fixing name and update version

viniciusvieira00 committed:
Merge pull request #37 from mdsreq-fga-unb/doc/30-cronograma-e-entregas

docs: change in delivery schedule and sprints

viniciusvieira00 committed:
Merge pull request #36 from mdsreq-fga-unb/doc/29-estratégias-de-esw

docs: add theoric references in the software engineering strategies document

BrunoBReis committed:
Merge pull request #38 from mdsreq-fga-unb/doc/32-cenario-atual

Doc/32 cenario atual

BrunoBReis committed:
docs: updating gitignore with .venv

BrunoBReis committed:
docs: updating politica commits to kabeb case

BrunoBReis committed:
docs: updating politica branches to kabeb case

BrunoBReis committed:
docs: updating and translating codigo conduta to kabeb case

BrunoBReis committed:
docs: simplify cronograma entregas

BrunoBReis committed:
docs: simplify interacao equipe cliente

BrunoBReis committed:
docs: simplify estrategias ESW

BrunoBReis committed:
docs: updating mkdocs yml file

BrunoBReis committed:
style: expand sprints

BrunoBReis committed:
docs: updating ferramentas de comunicacao

BrunoBReis committed:
docs: small fix to branch name

viniciusvieira00 committed:
docs:removed unused content

viniciusvieira00 committed:
docs: remove duplicate and unused content

viniciusvieira00 committed:
docs: standardizing version control of documents

BrunoBReis committed:
Merge pull request #40 from mdsreq-fga-unb/doc/39-pequenas_alteracoes

Doc/39 pequenas alteracoes

fabioaletorres committed:
Atualização da composição da equipe

PauloFilho2 committed:
docs: update segmentacao-do-cliente

PauloFilho2 committed:
Merge pull request #42 from mdsreq-fga-unb/34-empresas-de-manutenção-automotiva-e-oficinas-informar-o-perfil-das-pessoas-dessas-empresas-que-interagem-com-a-oficina-e-irão-interagir-com-o-produto

docs: update segmentacao-do-cliente

BrunoBReis commented:
@fabioaletorres esse pull request está errado. Não vamos colocar para a main toda a branch de docs. Onde foi feito as alterações?

BrunoBReis committed:
improve: updating gitignore file

BrunoBReis committed:
docs: creating engenharia-requisitos

BrunoBReis committed:
docs: updating name

BrunoBReis committed:
docs: adding version

BrunoBReis committed:
docs: updating deep

BrunoBReis committed:
Merge pull request #49 from mdsreq-fga-unb/doc/44-engenharia-de-requisitos

Doc/44 engenharia de requisitos

Stain19 committed:
Adiocionando provaveis causa nao resolviviveis apenas com o software ao arquivo solucao proposta

BrunoBReis committed:
docs: updating mkdocs yml

BrunoBReis committed:
docs: adding lista requisitos

Co-authored-by: Fabio Torres <fabioalessandrotorres@gmail.com>

viniciusvieira00 committed:
docs: add product backlog documentation

BrunoBReis committed:
docs: updating mkdocs yml

BrunoBReis committed:
docs: creating dor-dod

fabioaletorres committed:
Merge pull request #54 from mdsreq-fga-unb/doc/45-lista-requisitos

Doc/45 lista requisitos

fabioaletorres committed:
atualização de cargos dos membros da equipe

BrunoBReis committed:
docs: small changes

Stain19 committed:
Adicionando versionamento do arquivo solucao-proposta.md

BrunoBReis committed:
docs: updating team

BrunoBReis committed:
Merge pull request #59 from mdsreq-fga-unb/hotfix/55-att-readme

atualização de cargos dos membros da equipe (docs)

BrunoBReis committed:
Merge pull request #52 from mdsreq-fga-unb/33-causas-nao-resolviveis-software

Adiciona seção ao documento para identificar causas não resolvíveis apenas com software

BrunoBReis committed:
docs: updating dor

viniciusvieira00 committed:
Merge pull request #56 from mdsreq-fga-unb/46-atualizar-a-visão-de-produto-e-projeto-iii

docs: add product backlog documentation

BrunoBReis committed:
docs: updating engenharia requisitos

BrunoBReis committed:
docs: removing unused file

BrunoBReis committed:
Merge branch 'docs' into doc/47-dor-dod

BrunoBReis committed:
Merge pull request #57 from mdsreq-fga-unb/doc/47-dor-dod

Doc/47 dor dod

viniciusvieira00 committed:
Merge pull request #63 from mdsreq-fga-unb/doc/62-estrategias-esw

Doc/62 estrategias esw

BrunoBReis committed:
docs: updating backlog

viniciusvieira00 committed:
Merge pull request #65 from mdsreq-fga-unb/doc/64-product-backlog

docs: updating backlog

Stain19 committed:
docs: added learned lessons for unity 2

viniciusvieira00 committed:
Merge pull request #68 from mdsreq-fga-unb/66-learned-lessons

docs: added learned lessons for unity 2

BrunoBReis committed:
docs: updating home

BrunoBReis committed:
docs: updating intereacao equipe cliente

BrunoBReis committed:
Merge pull request #70 from mdsreq-fga-unb/doc/69-interacao-equipe-cliente

Doc/69 interacao equipe cliente

viniciusvieira00 committed:
docs: added second presentation of the project

viniciusvieira00 committed:
Merge pull request #71 from mdsreq-fga-unb/doc/48-lessons-learned

Doc/48-lessons-learned

Stain19 committed:
docs: add issue model

Co-authored-by: Vinicius Vieira <64455111+viniciusvieira00@users.noreply.github.com>

PauloFilho2 committed:
docs: updating backlog

BrunoBReis committed:
Merge pull request #83 from mdsreq-fga-unb/75-backlog

docs: updating backlog

arthur-heleno committed:
Update product-backlog

Atualizar o arquivo product-backlog com as escalas numéricas de todos os critérios de priorização

arthur-heleno committed:
Update product-backlog

Atualizar prioridades na matriz com os nomes.

arthur-heleno committed:
Update product-backlog

Atualizar com as cores

BrunoBReis committed:
Merge pull request #87 from mdsreq-fga-unb/docs-76-product-backlog

Update product-backlog

PauloFilho2 committed:
docs: Updating requisitos funcionais

PauloFilho2 committed:
docs: Updating backlog and requisitos funcionais

arthur-heleno committed:
Update engenharia-requisitos

Adição de Workshop em Análise e Consenso do Planejamento da Sprint e Atualização de Ver/Val da Execução da Sprint com DEEP e DoR

BrunoBReis committed:
Merge pull request #90 from mdsreq-fga-unb/docs-72-engenharia-requisitos

Update engenharia-requisitos

PauloFilho2 committed:
docs: adding prioridade in the backlog table

BrunoBReis committed:
docs: updating yml to estudo-de-caso

BrunoBReis committed:
docs: creating estudo-de-caso file

Stain19 committed:
Merge pull request #93 from mdsreq-fga-unb/doc/92-estudo-de-caso-usm

Doc/92 estudo de caso usm

arthur-heleno committed:
Update licoes-aprendidas

Adição das lições aprendidas na Unidade 3

BrunoBReis committed:
docs: updating mkdocs yml with estudo-de-caso-pbb

BrunoBReis committed:
docs: creating file estudo-de-caso-pbb

fabioaletorres committed:
Merge pull request #98 from mdsreq-fga-unb/docs-95-licoes-aprendidas

Update licoes-aprendidas

BrunoBReis committed:
docs: updating small sytle

fabioaletorres committed:
Merge pull request #99 from mdsreq-fga-unb/doc/94-estudo-de-caso-pbb

Doc/94 estudo de caso pbb

BrunoBReis committed:
Merge pull request #97 from mdsreq-fga-unb/73-lista-de-requisitos-funcionais

BrunoBReis committed:
docs: adding video 3

arthur-heleno committed:
Merge pull request #101 from mdsreq-fga-unb/doc/100-apresentacao-3

docs: adding video 3

viniciusvieira00 committed:
fix(docs): resolve wrong url to prensentation 3

viniciusvieira00 committed:
fix(docs): resolve wrong portuguese

BrunoBReis committed:
fix: removing lost file

BrunoBReis committed:
git: updating gitignore with nets files

BrunoBReis committed:
build: creating a makefile for quick docs build

BrunoBReis committed:
docs: updating mkdocs sidebar

BrunoBReis committed:
docs: removing basd images

BrunoBReis committed:
docs: updating bdd to new format

BrunoBReis committed:
docs: removing big file

BrunoBReis committed:
docs: creating criterios de aceitacao

BrunoBReis committed:
docs: creating estudo de caso

BrunoBReis committed:
docs: creating lista de historia de usuario

BrunoBReis committed:
docs: creating problemas identificados

BrunoBReis committed:
docs: adding new images

BrunoBReis committed:
docs: update bdd

BrunoBReis committed:
Merge pull request #120 from mdsreq-fga-unb/doc/105-pbb-bdd

Doc/105 pbb bdd

Stain19 committed:
Merge pull request #110 from mdsreq-fga-unb:doc/109-unidade3-video

Documentation: 109-unidade3-video

fabioaletorres committed:
Ajuste ortográfico e atualização da numeração das US

fabioaletorres committed:
Vinculação das US com as sprints correspondentes

BrunoBReis committed:
Merge pull request #122 from mdsreq-fga-unb/doc/att-ortografica-US

Ajuste ortográfico e atualização da numeração das US

BrunoBReis committed:
docs: updating mkdocs

BrunoBReis committed:
docs: removing bad images

BrunoBReis committed:
docs: updating usm to new file

BrunoBReis committed:
docs: adding new images

BrunoBReis committed:
docs: adding criterios de aceitacao

BrunoBReis committed:
docs: adding historias de usuario

BrunoBReis committed:
docs: adding persona

BrunoBReis committed:
Merge pull request #125 from mdsreq-fga-unb/doc/124-atualizando-o-usm

Doc/124 atualizando o usm

arthur-heleno committed:
Create folder docs/estudo-de-caso-uc

Adicionei a pasta estudo-de-caso-uc e o arquivo de template em markdown.

arthur-heleno committed:
Update template-especificacao-caso-de-uso.md

Atualizei o nome do arquivo para kabeb-case, mudei a posição do histórico de revisão para o final do arquivo e adicionei subtítulos.

arthur-heleno committed:
Update template-especificacao-caso-de-uso

Stain19 committed:
Merge pull request #123 from mdsreq-fga-unb/doc/111-att-cronograma

Vinculação das US com as sprints correspondentes

arthur-heleno committed:
Update template-especificacao-caso-de-uso.md

Adicionei Regras de Negócio e Atores.

arthur-heleno committed:
Update mkdocs.yml

Efetivamente adicionando ao gitpages o estudo de caso de casos de uso.

BrunoBReis committed:
Merge pull request #135 from mdsreq-fga-unb/131-docs/estudo-de-caso-uc

131-docs/estudo-de-caso-uc/template-especificacao-caso-de-uso

BrunoBReis committed:
feat: updating mkdocs

BrunoBReis committed:
feat: caso de uso 01

BrunoBReis committed:
feat: creating the introduction

BrunoBReis committed:
feat: updating links

arthur-heleno committed:
Merge pull request #141 from mdsreq-fga-unb/doc/140-atualizando-template

Doc/140 atualizando template

arthur-heleno committed:
Add caso-de-uso-02.md

arthur-heleno committed:
Update caso-de-uso-02.md

arthur-heleno committed:
Update caso-de-uso-02.md

BrunoBReis committed:
style: changing FE01

arthur-heleno committed:
Merge pull request #143 from mdsreq-fga-unb/hotfix/formatacao-caso-de-uso

style: changing FE01

arthur-heleno committed:
Update caso-de-uso-02.md

Atualizar os passos dos FEs

arthur-heleno committed:
Update caso-de-uso-02.md

Retirar negrito e indices

arthur-heleno committed:
Update introducao.md

arthur-heleno committed:
Merge pull request #142 from mdsreq-fga-unb/129-docs/estudo-de-caso-uc/caso-de-uso-02.md

Add caso-de-uso-02.md

PauloFilho2 committed:
add licoes aprendidas unidade 4

arthur-heleno committed:
Update cronograma-entregas.md

Ajustes na formatação

arthur-heleno committed:
Merge pull request #147 from mdsreq-fga-unb/144-docs/cronograma-entregas.md

Update cronograma-entregas.md

BrunoBReis committed:
Merge pull request #146 from mdsreq-fga-unb/130-lições-aprendidas

add licoes aprendidas unidade 4

BrunoBReis committed:
docs: updating yml

BrunoBReis committed:
docs: creating us4

BrunoBReis committed:
docs: creating us6

BrunoBReis committed:
docs: creating uc5

arthur-heleno committed:
Merge pull request #157 from mdsreq-fga-unb/doc/adicionando-template-de-caso-de-uso

Doc/adicionando template de caso de uso

PauloFilho2 committed:
add caso de uso 03

PauloFilho2 committed:
alterando mkdocs

PauloFilho2 committed:
atualizando caso de uso 03

PauloFilho2 committed:
atualizando os links do caso de uso 03

BrunoBReis committed:
Merge branch 'docs' into doc/129-caso-de-uso-03

PauloFilho2 committed:
Merge pull request #158 from mdsreq-fga-unb/doc/129-caso-de-uso-03

Doc/129 caso de uso 03
